<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCorrPTWidget</class>
 <widget class="QMainWindow" name="TCorrPTWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="keyWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="keySplitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTextEdit" name="edgeQuery"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="bsonWidget" native="true">
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>5</number>
        </property>
        <item row="0" column="1">
         <widget class="QComboBox" name="typeBox"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Schema</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="listsWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="lstSplitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="inWidget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="labelOut">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Outgoing Vertex - double-click to select its Id into Edge _OutV field</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="outWidget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="labelIn">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Incoming Vertex - double-click to select its Id into Edge _InV field</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDataBase">
    <property name="title">
     <string>&amp;Database</string>
    </property>
    <addaction name="actionSearch"/>
    <addaction name="actionSearch_Results"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionImport_CfgFile"/>
    <addaction name="actionExport_CfgFile"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionContents"/>
    <addaction name="separator"/>
    <addaction name="actionVersion"/>
    <addaction name="actionAuthors"/>
    <addaction name="actionLicense"/>
    <addaction name="separator"/>
    <addaction name="action_Help_About"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionChange_Property_list"/>
    <addaction name="separator"/>
    <addaction name="actionRealloc_T"/>
    <addaction name="actionReset_T"/>
    <addaction name="separator"/>
    <addaction name="actionRealloc_P"/>
    <addaction name="actionReset_P"/>
    <addaction name="separator"/>
    <addaction name="actionCopy_Field_Path"/>
    <addaction name="actionCopy_Field"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionKeep_Data_Fields_Expanded"/>
    <addaction name="action_Show_comments"/>
    <addaction name="action_Display_enums"/>
    <addaction name="action_Edit_id"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuApplication">
    <property name="title">
     <string>Applicatio&amp;n</string>
    </property>
    <addaction name="actionAction_1"/>
    <addaction name="separator"/>
    <addaction name="actionAction_2"/>
   </widget>
   <widget class="QMenu" name="menu_Calc">
    <property name="title">
     <string>&amp;Calc</string>
    </property>
    <addaction name="actionCalculate_Properties"/>
    <addaction name="actionCalculate_TP"/>
   </widget>
   <widget class="QMenu" name="menuResults">
    <property name="title">
     <string>&amp;Results</string>
    </property>
    <addaction name="actionShow_Results"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuDataBase"/>
   <addaction name="menu_Calc"/>
   <addaction name="menuResults"/>
   <addaction name="menu_View"/>
   <addaction name="menuHelp"/>
   <addaction name="menuApplication"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSearch"/>
   <addaction name="actionPrevious_Record"/>
   <addaction name="actionNext_Record"/>
   <addaction name="separator"/>
   <addaction name="actionChange_Property_list"/>
   <addaction name="separator"/>
   <addaction name="actionCalculate_Properties"/>
   <addaction name="actionCalculate_TP"/>
   <addaction name="actionShow_Results"/>
  </widget>
  <widget class="QToolBar" name="nameToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Help_About"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionImport_CfgFile">
   <property name="text">
    <string>&amp;Read Config Data from  File...</string>
   </property>
  </action>
  <action name="actionExport_CfgFile">
   <property name="text">
    <string>&amp;Write Config Data to  File ...</string>
   </property>
  </action>
  <action name="action_Help_About">
   <property name="icon">
    <iconset resource="tcorrpt.qrc">
     <normaloff>:/menu/Icons/QuestionMessageIcon24.png</normaloff>:/menu/Icons/QuestionMessageIcon24.png</iconset>
   </property>
   <property name="text">
    <string> Help &amp;About</string>
   </property>
   <property name="toolTip">
    <string>Show Help VIewer ...</string>
   </property>
  </action>
  <action name="actionAuthors">
   <property name="text">
    <string>Au&amp;thors ...</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>&amp;License ...</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="text">
    <string>Help &amp;Contents ...</string>
   </property>
  </action>
  <action name="action_Show_comments">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Display Comments to Data Fields</string>
   </property>
  </action>
  <action name="action_Display_enums">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Show Enum Names Instead of Values</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="tcorrpt.qrc">
     <normaloff>:/menu/Icons/SelectDisplayRecordIcon24.png</normaloff>:/menu/Icons/SelectDisplayRecordIcon24.png</iconset>
   </property>
   <property name="text">
    <string>Search: Create New &amp;Query ...</string>
   </property>
  </action>
  <action name="actionSearch_Results">
   <property name="text">
    <string>E&amp;xplore Search Results Table ...</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>&amp;Version ...</string>
   </property>
  </action>
  <action name="actionAction_1">
   <property name="text">
    <string>&amp;Action 1 ...</string>
   </property>
  </action>
  <action name="actionAction_2">
   <property name="text">
    <string>Action &amp;2 ...</string>
   </property>
  </action>
  <action name="action_Edit_id">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Edit _id of DB (Document) Records </string>
   </property>
  </action>
  <action name="actionKeep_Data_Fields_Expanded">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Keep Data Fields Expanded</string>
   </property>
  </action>
  <action name="actionNext_Record">
   <property name="icon">
    <iconset resource="tcorrpt.qrc">
     <normaloff>:/menu/Icons/DisplayNextRecordIcon24.png</normaloff>:/menu/Icons/DisplayNextRecordIcon24.png</iconset>
   </property>
   <property name="text">
    <string>Next Record</string>
   </property>
   <property name="toolTip">
    <string>Next Record</string>
   </property>
  </action>
  <action name="actionPrevious_Record">
   <property name="icon">
    <iconset resource="tcorrpt.qrc">
     <normaloff>:/menu/Icons/DisplayPrevRecordIcon24.png</normaloff>:/menu/Icons/DisplayPrevRecordIcon24.png</iconset>
   </property>
   <property name="text">
    <string>Previous Record</string>
   </property>
  </action>
  <action name="actionCopy_Field_Path">
   <property name="text">
    <string>Copy Field &amp;Path</string>
   </property>
  </action>
  <action name="actionCopy_Field">
   <property name="text">
    <string>Copy &amp;Field</string>
   </property>
  </action>
  <action name="actionCalculate_Properties">
   <property name="icon">
    <iconset resource="tcorrpt.qrc">
     <normaloff>:/menu/Icons/DatabaseFunctionIcon24.png</normaloff>:/menu/Icons/DatabaseFunctionIcon24.png</iconset>
   </property>
   <property name="text">
    <string>Calculate &amp;Properties</string>
   </property>
  </action>
  <action name="actionCalculate_TP">
   <property name="icon">
    <iconset resource="tcorrpt.qrc">
     <normaloff>:/menu/Icons/RTparmModuleIcon24.png</normaloff>:/menu/Icons/RTparmModuleIcon24.png</iconset>
   </property>
   <property name="text">
    <string>Calculate TP</string>
   </property>
  </action>
  <action name="actionShow_Results">
   <property name="icon">
    <iconset resource="tcorrpt.qrc">
     <normaloff>:/menu/Icons/ShowGEMResultsIcon24.png</normaloff>:/menu/Icons/ShowGEMResultsIcon24.png</iconset>
   </property>
   <property name="text">
    <string>Show Results</string>
   </property>
  </action>
  <action name="actionChange_Property_list">
   <property name="icon">
    <iconset resource="tcorrpt.qrc">
     <normaloff>:/menu/Icons/DocumentsPileIcon24.png</normaloff>:/menu/Icons/DocumentsPileIcon24.png</iconset>
   </property>
   <property name="text">
    <string>Change Property list</string>
   </property>
  </action>
  <action name="actionRealloc_T">
   <property name="text">
    <string>Realloc T</string>
   </property>
  </action>
  <action name="actionReset_T">
   <property name="text">
    <string>Reset T</string>
   </property>
  </action>
  <action name="actionRealloc_P">
   <property name="text">
    <string>Realloc P</string>
   </property>
  </action>
  <action name="actionReset_P">
   <property name="text">
    <string>Reset P</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="tcorrpt.qrc"/>
 </resources>
 <connections/>
</ui>
